front/src/routes/integration/all/external-integration/install-from-github/style.css

```
@@ -32,6 +32,17 @@
   color: inherit;
 }
 
+.errorDetails {
+  margin-bottom: 0;
+  padding-left: 1.25rem;
+  font-family: SFMono-Regular, Menlo, Consolas, monospace;
+  font-size: 0.75rem;
+  word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)